src/modules/account/submodule/multisig/views/ViewMultisigHome.vue

```diff
@@ -13,7 +13,7 @@
       <div class='border p-4 my-3 '>
        <div class="flex flex-col gap-2">
         <div v-for="(cosigner,index) in cosignerAccountsList" :key="index">
-            <div class="border w-full cursor-pointer rounded-md p-3" @click="navigate(cosigner.address)">
+            <div class="border w-full cursor-pointer rounded-md p-3" @click="navigate(cosigner.address);explorerLink(cosigner.address)">
```

Comment:
why navigate and open pop up windows with single click ? Just navigate and open the pop up after navigate